terraform-beyond-layer0/example-1/modules/guestbook_service/main.tf

```diff
@@ -1,9 +1,5 @@
-#
```

Comment:
We don't have any walkthroughs that focus on writing good terraform modules I believe. At least, not to my knowledge. Might have to pair up on this somehow; need further elaboration on "releasing" terraform modules.